knexfile.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    test: {
```

Comment:
Duplication. You can have one config but different environment variables for test and production env.